CONTRIBUTING.md

```diff
@@ -66,6 +66,18 @@ If you would like to add new capabilities to the product as described in **B)**,
 
     Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.
 
+## Get To Know System App
+
+There are several chapters on Microsoft Learn that explains several aspects of the System app.  
+Here are some:
+
+- [Module Architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
+- [Get started with modules in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
+- [Set Up an Environment for Developing a Module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
+- [Create a new module in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
+- [Create a .NET Wrapper Module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
+- [Change a module in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Comment:
I'd stick to the TOC view:
![image](https://github.com/microsoft/BCApps/assets/34504100/89999245-0d88-415a-b7a7-d5c61cf8280c)


```suggestion
- [Module architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Get started with modules](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Set up your development environment](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
- [Create a new module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
- [Create a .NET wrapper module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
- [Change a module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Reply:
@NKarolak  You mean like this? 😉
![image](https://github.com/microsoft/BCApps/assets/34970096/994ea979-7615-4e15-ac09-09861440ce97)

Reply:
It's only about the casing: Please look again at my code suggestion above, the green part that did not make it into your screenshot, except a thin green bottom line ;-)

Reply:
Crap, you are right again! 😊